src/helpers/amoc_xmpp.erl

```diff
@@ -40,6 +45,13 @@ connect_or_exit(Spec) ->
             exit(connection_failed)
     end.
 
+maybe_use_legacy_tls(Spec) ->
+    case amoc_config:get(legacy_tls) of
+        false -> Spec;
+        true ->
+            lists:keystore(ssl, 1, Spec, {ssl, true})
```

Comment:
Could you please or remove new line here or add in like 50? To make the block look the same.